src/components/Container.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { theme } from '../constants';
+
+const { SIZES } = theme;
+
+const Container = ({ children }) => {
+    return (
+        <View style={styles.container}>
+            { children }
+        </View>
+    )
```

Comment:
The CodeClimate tells them that you can try refactoring here 😊 

![image](https://user-images.githubusercontent.com/69128578/114465499-cf53fc00-9bac-11eb-9566-1e36bfb573a0.png)